client/src/pages/Profile/ProfileDetail/ProfileDetail.tsx

```diff
@@ -4,24 +4,33 @@ import { Grid, Box, CircularProgress } from '@mui/material';
 import Layout from '../../Layout/Layout';
 import ProfileDetailCard from './ProfileDetailCard/ProfileDetailCard';
 import BookingCard from './BookingCard/BookingCard';
-import { sampleData, Sitter } from './sampleData';
+import { Profile } from '../../../interface/Profile';
+import { getProfileDetail } from '../../../helpers/APICalls/profiles';
 
 const ProfileDetail = (): JSX.Element => {
   const params: { sitterId: string } = useParams();
   const { sitterId } = params;
 
-  const [sitter, setSitter] = useState<Sitter | null | undefined>(null);
-
-  const getSitterInfo = (sitterId: string) => {
-    return sampleData.find((sitter: Sitter) => sitter.sitterId === sitterId);
-  };
+  const [sitter, setSitter] = useState<Profile | null | undefined>(null);
 
   useEffect(() => {
-    const sitter = getSitterInfo(sitterId);
-    setSitter(sitter);
+    try {
+      const fetchProfileDetail = async () => {
+        const fetchedProfileDetail = await getProfileDetail(sitterId);
+        setSitter(fetchedProfileDetail);
+      };
+      fetchProfileDetail();
+    } catch (error) {
+      console.log(error);
```

Comment:
Remove console log please

Reply:
Applied

Reply:
@jstyle5 You can update the snackbar here rather than logging the error to the console

Reply:
Applied